R/structural.properties.R

```diff
@@ -2310,9 +2328,11 @@ bfs <- function(
 #'
 dfs <- function(graph, root, mode = c("out", "in", "all", "total"),
```

Comment:
Use the opportunity to add an ellipsis after `mode` ? Also with `bfs()` ?

Reply:
makes sense!